### PR TITLE
May 21 Blog Post

### PR DESCRIPTION
**Pull request by DrewML**
cc @miketaylr

### CODE REVIEW

**Comment by beaktor** [comment]
Love the deprecating humor. Also, RIP Spredfast in that template.

File: _includes/give-em-the-business.html [hunk 1]
```diff
@@ -14,6 +14,10 @@
     <li>
       Where: <a href="http://thegingerman.com/austin/">The Gingerman Austin &mdash; 301 Lavaca Street &mdash; Austin, TX 78701</a>
     </li>
+  {% elseif include.location == "indeed" %}
```
Comment (aaronj1335) [inline _includes/give-em-the-business.html]:
```suggestion
  {% elsif include.location == "indeed" %}
```

File: _posts/2019-05-21-meetup.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
```
Comment (aaronj1335) [inline _posts/2019-05-21-meetup.md]: this will need to be named `_posts/2019-05-20-meetup.md`

File: _posts/2019-05-21-meetup.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<div class="media-object speaker-bio">
+  <a href="{{ twiturl }}">
+    <img alt="{{ speakr }} @miketaylr on Twitter"
+      src="https://pbs.twimg.com/profile_images/1042621997/mikeyyyy_400x400.png" />
+  </a>
+  <div>
+  <a href="{{ twiturl }}"><strong>{{ speakr }}</strong></a>
+
+  Mike Taylor is an Engineering Manager on the Web Compatibility Team at Mozilla. Come learn more about him, as the author of this blog post does not know that much!
```
Comment (aaronj1335) [inline _posts/2019-05-21-meetup.md]:
```suggestion
  Mike Taylor, who has been called the Carson Daly of Javascript, is an Engineering Manager on the Web Compatibility Team at Mozilla. Come learn more about him, as the author of this blog post does not know that much!
```

**Comment by aaronj1335** [comment]
sorry this is so painful... i'll try to write up how to preview these and maybe a quick travis check

**Comment by aaronj1335** [comment]
ok i added instructions and tests that'll break the travis build if the site doesn't build or there's a post in the future.

between the python, ruby, and shell used to build this javascript meetup site, the level of over-engineering here will probably get me a strongly-worded (and well-deserved) tweet from uncle @davatron5000

File: _posts/2019-05-21-meetup.md (hunk 3 above)
Comment (miketaylr) [reply _posts/2019-05-21-meetup.md]: lol

**Comment by davatron5000** [comment]
This is how y'all invite me to the meetup? This. Here? Some new kind of rage-inducing invite. 

If y'all need help porting the meetup site over to something less... uh... "technical"... let me know.

**Comment by aaronj1335** [comment]
it could be worse.... we could have signed you up to speak!

**Comment by davatron5000** [comment]
So what I'm hearing is you'd like to play around with Erlang. Got it. I'll start porting things over. Wanna just give me admin creds? I think it'll go a lot faster.

**Comment by miketaylr** [comment]
very excited for the @davatron5000 AMP html re-write

**Pull request merged**